dist/Client.js

```diff
@@ -56,6 +56,7 @@ class ValidationError extends Error {
 async function checkResponse(response) {
   if (!response.ok) {
     const text = await response.text();
+    console.error('Error checking response', text);
```

Comment:
We are already throwing an error, the `console.error` isn't necessary

Reply:
Checked